src/normalize/glossary.ts

```diff
@@ -149,6 +149,21 @@ export default function normalizeGlossary(editor: Editor, entry: NodeEntry): boo
     }
 
     if (DefinitionTerm.isDefinitionTerm(node)) {
+        // check if there is Term somewhere in DefinitionTerm.
+        // If it is, unwarps it
+        const removeTerm = (nodePathToCheck: Path) => {
+            for (const [child, childPath] of Node.children(editor, nodePathToCheck)) {
+                if (!Text.isText(child) && Node.children(editor, childPath)) {
+                    removeTerm(childPath)
+                }
+                if (Term.isTerm(child)) {
+                    Transforms.unwrapNodes(editor, { at: childPath })
+                    return true
+                }
+            }
+        }
+        removeTerm(path)
```

Comment:
You shouldn't do it like this, since all children of `node` will be normalized. Instead in normalization for `Term` check, that it's not nested in a `DefinitionTerm`:

```ts
if (Editor.above(editor, { at: path, match: DefinitionTerm.isDefinitionTerm }) != null) {
    // is descendant of `DefinitionTerm`
}
```